store/errors.go

```diff
@@ -183,28 +191,38 @@ func (e SnapActionError) Error() string {
 			header = "cannot install or download:"
 		}
 	}
-	es := []string{header}
 
-	for name, e := range e.Refresh {
-		es = append(es, fmt.Sprintf("snap %q: %v", name, e))
-	}
+	// reverse the maps, as the common case is that they're all the same
+	reversed := map[string][]string{}
+	revkeys := []string{} // poorman's ordered map
```

Comment:
Having `strutil.OrderedMap` do we need to do a poor man's version?

Reply:
I looked at it, and it's not friendly about modifying keys (and it's a `map[string]string` which means I don't get to sort the snap names)